features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/InfoBar.jsx

```diff
@@ -365,7 +366,11 @@ class InfoBar extends React.Component {
                     </Collapse>
                 )}
                 <div className={classes.infoContentBottom}>
-                    <div className={classes.contentWrapper} onClick={() => this.toggleOverview()}>
+                    <div
+                        className={classes.contentWrapper}
+                        onClick={() => this.toggleOverview()}
```

Comment:
Why are we nesting the `this.toggleOverview` inside a lambda function?

if you need to preserve the `this` object, you could bind `this` to `this.toggleOverview` function in  the constructor

Reply:
fixed  https://github.com/wso2/carbon-apimgt/pull/6787/commits/b09339ab96cfdcc35f13e351833f406efad26ead